app/controllers/users_controller.rb

```diff
@@ -19,6 +19,14 @@ def new
   def edit
   end
 
+  def orders
+    @user = User.find(session[:user_id])
+  end
+
+  def line_item
+    @user = User.find(session[:user_id])
+  end
+
```

Comment:
1. `@user = User.find(session[:user_id])` is repeated code here so extract this out at better place where multiple controllers can access it
2. Why not here only load order and line_item so it will not be responsibilty of view to load it